front/src/applications/operationalStudies/consts.ts

```diff
@@ -71,7 +71,11 @@ export const MANAGE_TRAIN_SCHEDULE_TYPES = Object.freeze({
   opendata: 'OPENDATA',
 });
 
-export const MODES = Object.freeze({
+interface MODESTypes {
```

Comment:
```suggestion
interface MODES_TYPES {
```
MODES_TYPES would be more readable